main.cpp

```diff
@@ -131,28 +179,28 @@ int main()
     //example of calling that function, storing the value, and passing it to ignoreUnused at the end of main()
     auto carRented = rentACar(6, 2); 
     
-    //1)
+    auto sum = addition(2, 4);
     
-    //2)
+    auto nextPath = pathFinder('C', 'B');
     
-    //3)
+    auto myGpsCoords = gpsFinder(36.056595f, -112.125092f);
     
-    //4)
+    playerCharacterMaker(35, 'D', 182, true);
     
-    //5)
+    climateAnalysis(82.2f, 76, true);
     
-    //6)
+    auto hiddenMessageKey = messageEncoder('X');
     
-    //7)
+    auto distanceInKilometers = lightyearConversion(70050);
     
-    //8)
+    auto atomicWeightSum = atomicMassCalculator(1.008, 'H', 19);

```

Comment:
`1.008` is a `double`.   The function wants a `float` passed as the first argument.
this is causing a warning in the program output
```
main.cpp:196:49: warning: implicit conversion loses floating-point precision:
      'double' to 'float' [-Wconversion]
    auto atomicWeightSum = atomicMassCalculator(1.008, 'H', 19);
```
- [ ] revise